src/styles/main.scss

```diff
@@ -0,0 +1,60 @@
+@import './utils/variables';
+
+body {
+  margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  padding: 10px;
+  flex-wrap: wrap;
+  gap: $base-gap;
+  width: $base-size * 7 + $base-gap * 6;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), calculations should be wrapped in parentheses to improve readability. Wrap the arithmetic operations in parentheses.